src/components/studies/StudyList.tsx

```diff
@@ -127,13 +129,24 @@ const StudySublist: FunctionComponent<StudySublistProps> = ({
   if (!displayStudies.length) {
     return <></>
   }
+
+  const sortStudies = (a: Study, b: Study, sortBy: String): number => {
+    const dateA = sortBy === 'CREATION' ? a.createdOn! : a.modifiedOn!
+    const dateB = sortBy === 'CREATION' ? b.createdOn! : b.modifiedOn!
+    return dateA >= dateB ? -1 : 1
+  }
+
+  if (status === 'DRAFT') {
```

Comment:
I think you ca replace custom sorting with lodash orderBy method https://lodash.com/docs/4.17.15#orderBy
We already have lodash `_` as a dependency